package com.ibm.collectionframework;

public class Exception1 
{
	
	public static void main(String[] args) 
	{
	throw new ArithmeticException("/ by zero hai");
	/*public void demo()
	{
		int a[]= {1,2,3,4,5,6};
		try
		{
			System.out.println(a[2]);
		} 
		catch(Exception e)
		{
			System.out.println("please enter the valid length...:"+a.length);
		}
		finally
		{
			System.out.println("finally will be executed");
		}*/
	/*
	 * public void fun() { System.out.println(1/0);//Arithmetic exception will
	 * occurs }
	 */
	}

}


